server/src/routes/api/v1/memesRouter.js

```diff
@@ -17,9 +17,10 @@ memesRouter.get("/", async (req, res) => {
 });
 
 memesRouter.get("/:id", async (req, res) => {
+  const { id } = req.params
   try {
-    const memeId = req.params.id;
-    const meme = await Meme.query().findById(memeId);
+    const meme = await Meme.query().findById(id);
+    meme.reviews = await meme.$relatedQuery("reviews");
```

Comment:
your reviews need to be serialized as well!

Reply:
Done